operator/export-vars.sh

```diff
@@ -26,13 +35,41 @@ images[VALIDATION_IMAGE]=quay.io/redhat-user-workloads/rh-mtv-1-tenant/forklift-
 
 images[VIRT_V2V_IMAGE]=quay.io/redhat-user-workloads/rh-mtv-1-tenant/forklift-operator/virt-v2v@sha256:17628ab1f836549509ad80c321f669441d3f15348a5ef151b69fbbcec03ccd30
 
+# For downstream, the names of the images change
+
+declare -A replacements
+
+replacements[API_IMAGE]=mtv-api-rhel9
+replacements[CONTROLLER_IMAGE]=mtv-controller-rhel8
+replacements[MUST_GATHER_IMAGE]=mtv-must-gather-rhel8
+replacements[OPENSTACK_POPULATOR_IMAGE]=mtv-openstack-populator-rhel9
+replacements[OPERATOR_IMAGE]=mtv-rhel8-operator
+replacements[OVA_PROVIDER_SERVER_IMAGE]=mtv-ova-provider-server-rhel9
+replacements[OVIRT_POPULATOR_IMAGE]=mtv-rhv-populator-rhel8
+replacements[POPULATOR_CONTROLLER_IMAGE]=mtv-populator-controller-rhel9
+replacements[UI_PLUGIN_IMAGE]=mtv-console-plugin-rhel9
+replacements[VALIDATION_IMAGE]=mtv-validation-rhel8
+replacements[VIRT_V2V_IMAGE]=mtv-virt-v2v-rhel8
+
 # Chage the repository of the images to match the repository they would be pushed
 # to during a release.
 
-for k in "${!images[@]}"; do
-    image="${images[$k]}"
-    new_image="${image/redhat-user-workloads\/rh-mtv-1-tenant/kubev2v}"
-    export "$k=$new_image"
-done
+if [ $stream == "upstream" ]; then
+    for k in "${!images[@]}"; do
+        image="${images[$k]}"
+        new_image="${image/quay.io\/redhat-user-workloads\/rh-mtv-1-tenant\/forklift-operator/kubev2v}"
+        export "$k=$new_image"
+    done
+elif [ $stream == "downstream" ]; then
+    for k in "${!images[@]}"; do
+        image="${images[$k]}"
+        new_image="${image/forklift-operator\/[^@]*@/forklift-operator\/${replacements[$k]}@}"
+        new_image="${new_image/quay.io\/redhat-user-workloads\/rh-mtv-1-tenant\/forklift-operator/registry.redhat.io\/migration-toolkit-virtualization}"
+        export "$k=$new_image"
```

Comment:
Wouldn't it be easier to manually create two files, like `image-names-downstream.env` and `image-names-upstream.env` which are manually maintained, and then just `source $FN.env` in the two branches?

Reply:
But you will lose the automation that creates the PRs with `quay.io/redhat-user-workloads` but then changes to the release registry when building the image. You will only have the nudges when actually releasing

Reply:
Could that automation work around two separate files? 
It would be easier if we will have 2 separate files directly showing the images which are in the bundle.

Reply:
@gbenhaim you started with this script. Any thoughts?

Reply:
@ernesgonzalez33 you can have 2 different files. Just make sure to reference the upstream file in the upstream dockerfile and the d/s file in the d/s dockerfile.

Reply:
Just discussed with @mnecas. For the downstream, they want the references to point directly to registry.redhat.io and be update with every release. Turning this PR back to draft to change that